app/assets/javascripts/role_table.es6

```diff
@@ -20,23 +20,23 @@ window.App.RoleTable = class RoleTable {
   init() {
     // Declare variables with defaults
     const {
-      tableSelector='.j-table__table',
-      tableSearchInputSelector='.j-table__search',
-      tableRowSelector='.j-table__row',
-      submitActionSelector= '.j-table__submit-changes',
+      tableSelector = '.j-table__table',
+      tableSearchInputSelector = '.j-table__search',
+      tableRowSelector = '.j-table__row',
+      submitActionSelector = '.j-table__submit-changes',
       tableCancelChange
     } = this.props
 
     // Init Datable
     this.table = $(tableSelector).DataTable({
       // scrollY: '55vh',
       scrollCollapse: true,
-      scrollX: true,
+      // scrollX: 'true',
```

Comment:
This is necessary so the table header isn't duplicated on the page.  See 
https://datatables.net/forums/discussion/42911/table-headers-getting-duplicated-above-table